deploy/playbooks/03_app.yml

```
@@ -3,26 +3,28 @@
 
   tasks:
     - name: Clone the repository to a specific version (to a temp location)
-      git:
+      ansible.builtin.git:
         repo: "{{ repository_url }}"
         dest: /tmp/src
-        accept_hostkey: yes
+        accept_hostkey: true
         version: "{{ app_version }}"
 
     - name: Build with a given commit hash
       # This will be stored in local registry, and available as version to docker-compose
       # where we can just reference correct version
-      shell: "cd /tmp/src && make docker/build V={{ app_version }}"
+      ansible.builtin.shell: "cd /tmp/src && make docker/build V={{ app_version }}"
```

```suggestion
      ansible.builtin.shell: 
        chdir: /tmp/src
        cmd: "/usr/bin/make docker/build V={{ app_version }}"
```
[builtin.shell](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/shell_module.html#ansible-collections-ansible-builtin-shell-module) does have a chdir. 
Does make need a shell? If not we could use ansible.builtin.command